docs/en/Deploy-skywalking-agent.md

```diff
@@ -39,5 +39,11 @@ CATALINA_OPTS="$CATALINA_OPTS -javaagent:/path/to/skywalking-agent/skywalking-ag
 - Tomcat 8
 Change the first line of `tomcat/bin/catalina.sh`.
 ```shell
-set "CATALINA_OPTS=-javaagent:E:\apache-tomcat-8.5.20\skywalking-agent\skywalking-agent.jar"
-```
\ No newline at end of file
+set "CATALINA_OPTS=-javaagent:/path/to/skywalking-agent/skywalking-agent.jar"
+```
+- Spring Boot JAR file
```

Comment:
This also fits all jar file start up. Recommend remove Spring, or just mention Spring in contents, not title.